src/validation/custom.validation.ts

```diff
@@ -29,13 +29,20 @@ export const checkInvalidCharactersIfRadioButtonSelected = (selected: boolean, e
   return true;
 };
 
-export const checkDateIsNotCompletelyEmpty = (errMsg: string, day: string = "", month: string = "", year: string = "") => {
+export const checkDateIsNotCompletelyEmptyWithErrorThrown = (errMsg: string, day: string = "", month: string = "", year: string = "") => {
```

Comment:
I think now having two functions with the same logic in apart from either throwing an error or returning false is not great. Better to have just the one method (keep the one below) which returns true or false and in those three (I think) cases where it's currently called and needs to throw the error, react to the 'false' value returned and do that there.

Reply:
Removed second function